react-table.js.org / react-form.js.org

tannerlinsley opened the pull request:
- There is reasonable content on the page (see: [No Content](https://github.com/js-org/dns.js.org/wiki/No-Content))
- I have read and accepted the [ToS](http://dns.js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But it seems that the ```CNAMES``` in your repo are not recognized. Because you are not using gh-pages branch, can you check that you set up everything correct following this: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/
I'll merge after we have "contact".

tannerlinsley commented:
Interesting. I am using githubs `/docs` option to build my github pages site, and I followed the directions for adding the CNAME file to the repo.

Looking through the repo settings more carefully though, it seems Github has now added an input box in the settings for a custom domain:
<img width="738" alt="screen shot 2017-01-18 at 9 19 41 am" src="https://cloud.githubusercontent.com/assets/5580297/22072302/3951a852-dd5f-11e6-9b10-c060946116ab.png">

I've never seen it until now, so let's see if it works!  I've updated it for both domains.

indus commented:
LGTM
Welcome  https://react-table.js.org / https://react-form.js.org
